test/Makefile.am

```diff
@@ -176,6 +178,8 @@ TESTS += units_unit
 TESTS += kinetics_reactive_scheme_unit.sh
 TESTS += sigma_bin_converter_unit
 TESTS += sigma_bin_converter_vec_unit
+TESTS += sigma_bin_converter_unit.sh
```

Comment:
This line should not be.  There is no such test nor file.  I have no idea of the history of it...